jsk_topic_tools/CMakeLists.txt

```diff
@@ -98,6 +98,8 @@ if (NOT $ENV{ROS_DISTRO} STREQUAL "indigo")
   add_rostest(test/test_hz_measure.test)
   add_rostest(test/test_block.test)
 endif(NOT $ENV{ROS_DISTRO} STREQUAL "indigo")
```

Comment:
I see.
So at this time, should I write `add_rostest` lines in side the if condition?